browser/src/control/Control.ColumnHeader.ts

```diff
@@ -143,6 +143,8 @@ export class ColumnHeader extends Header {
 		this.context.strokeStyle = this._borderColor;
 		this.context.lineWidth = app.dpiScale;
 		this.context.strokeRect(startX - 0.5, 0.5, entry.size, this.size[1]);
+
+        this._map._contextMenu.stopRightMouseUpEvent();
```

Comment:
this is repeated a lot of times and always adds new event listeners - causing to clow down calc with every time a bit...